index.html

```diff
@@ -15,7 +15,10 @@
         <main class="main_container">
 
             <section class= "achievements"> 
-                <!-- Изменения, например -->
+                        <img src="images/achievements1.png" class="image_scale"></img> 
+                        <img src="images/achievements2.png" class="image_scale"></img>
+                        <img src="images/achievements3.png" class="image_scale"></img>
+                        <img src="images/achievements4.png" class="image_scale"></img>
             </section>
 
             <h2 class="headlight font_bold_32">Our Services</h2>
```

Comment:
You missed suTitle "Early stage company? Eligible applicants get all of our products for just $49 a month."
I recommend to use code from our previous template:
      <h2>Our Services</h2>
      <span>Early stage company? Eligible applicants get all of our products for just $49 a month.</span>